tensorflow_datasets/structured/movielens_parsing_test.py

```diff
@@ -0,0 +1,507 @@
+# Copyright 2020 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for movielens_parsing."""
+
+import os
+
+import tensorflow_datasets.public_api as tfds
+from tensorflow_datasets.structured import movielens_parsing
+
+class MovieLensUtilsTest(tfds.testing.TestCase):
+  """Tests for helper functions in movielens_parsing"""
```

Comment:
Nit: add period.

Reply:
Fixed in ad8a410399cac16e254f4992ec7519ff46b14a16.